mountinfo/mountinfo_linux.go

```diff
@@ -0,0 +1,95 @@
+// +build linux
+
+package mountinfo
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+const (
+	/* 36 35 98:0 /mnt1 /mnt2 rw,noatime master:1 - ext3 /dev/root rw,errors=continue
```

Comment:
Please link to the place you copypasted this from: https://www.kernel.org/doc/Documentation/filesystems/proc.txt.

Also, please be mindful of copy pasting from other sources without attribution.